lldb/source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -2095,25 +2095,31 @@ void SwiftLanguageRuntime::DidFinishExecutingUserExpression(
 
 bool SwiftLanguageRuntime::IsABIStable() { FORWARD(IsABIStable); }
 
-struct AsyncUnwindRegisters {
```

Comment:
Should this be in an anonymous namespace?

Reply:
done